shared/constants/network.js

```diff
@@ -4,6 +4,9 @@ export const KOVAN = 'kovan';
 export const MAINNET = 'mainnet';
 export const GOERLI = 'goerli';
 export const NETWORK_TYPE_RPC = 'rpc';
+export const ETHEREUM = 'ethereum';
```

Comment:
could these three constants be moved to `shared/constants/swaps`?

I believe they will only be used in swaps, because we already use `'mainnet'` as the name for `'ethereum'`, and for all custom networks they are identified by chainId.

Reply:
Sure, will move them there.